src/_scss/pages/about/header/header.scss

```diff
@@ -0,0 +1 @@
+@import "components/pageTitleBar/pageTitleBar";
```

Comment:
Why do this? Why not just do this import in the parent file and delete this file?

Reply:
Copied CSS from Agency - will remove